simhash/__init__.py

```diff
@@ -107,11 +107,8 @@ def build_by_features(self, features):
                 w = f[1]
             for i in range(self.f):
                 v[i] += w if h & masks[i] else -w
-        ans = 0
-        for i in range(self.f):
-            if v[i] > 0:
-                ans |= masks[i]
-        self.value = ans
+        binary_str = ''.join(['0' if i <= 0 else '1' for i in v])
```

Comment:
Can you make it in reverse order here?
One way is `binary_str = ''.join(['0' if i <= 0 else '1' for i in reversed(v)])`
I just wanted to make it backward compatible.

Or maybe we need to bump up the version to `2.0` since this is a breaking change?

Which way do you think it's better?